packages/components-angular/package.json

```diff
@@ -41,7 +41,7 @@
     "angular-imask": "6.4.2",
     "imask": "^6.4.3",
     "rxjs": "^7.8.0",
-    "tslib": "^2.4.1",
+    "tslib": "^2.5.0",
```

Comment:
I noticed that we have `"tslib": "^2.0.1"` in `/packages/components-angular/projects/angular-wrapper/package.json`. Should we update it there too?

Reply:
They are upgraded as we can see in the yarn.lock. It seems they got previously upgraded via deletion of yarn.lock thats why its not mirrored in the package.json.